src/app/views/river-index/components/nwi-map.vue

```diff
@@ -36,9 +36,12 @@ import { basemapToggleMixin } from '@/app/global/mixins'
 import { mapState } from 'vuex'
 import {
   laravelDeploy,
-  nwiTileServer
+  nwiTileServer,
+  isAmplifyDeploy
 } from '@/app/environment'
 
+let tileserverToken = null;
```

Comment:
why define this here instead of just setting `tileserverToken: null` in `data`?

Reply:
My goal was to have it persist as much as possible across different mounts of this component. I would prefer that if we mount the map multiple times (or if we navigate forward and backward to the map) we don't need to re-fetch the token.

That being said, I may be misunderstanding the vue lifecycle and how that works!